lib/common/resources/otel-collector-config-new.yml

```diff
@@ -9,7 +9,14 @@ metadata:
   namespace: "{{namespace}}"
```

Comment:
remove this file `[otel-collector-config-new.yml](https://github.com/aws-observability/cdk-aws-observability-accelerator/pull/106/files#diff-553fe97b13846f964786cf995d0c66df9dc3d37b48db81e81d0977dc00fb834a)` always from checkin. Or please add to gitignore.

Reply:
done